src/applications/vaos/components/PastAppointmentsList.jsx

```diff
@@ -41,6 +42,15 @@ export class PastAppointmentsList extends React.Component {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    if (
+      prevProps.appointments.pastStatus === 'loading' &&
```

Comment:
We should use the `FETCH_STATUS` constant from utils/constants instead of the strings here.